daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-patterns.md

```diff
@@ -143,35 +143,33 @@ The Dapr workflow HTTP API supports the asynchronous request-reply pattern out-o
 The following `curl` commands illustrate how the workflow APIs support this pattern.
 
 ```bash
-curl -X POST http://localhost:3500/v1.0-alpha1/workflows/dapr/OrderProcessingWorkflow/12345678/start -d '{"input":{"Name":"Paperclips","Quantity":1,"TotalCost":9.95}}'
+curl -X POST http://localhost:3500/v1.0-alpha1/workflows/dapr/OrderProcessingWorkflow/start?instanceID=12345678 -d '{"Name":"Paperclips","Quantity":1,"TotalCost":9.95}'
```

Comment:
Q: If the management API is called start, why is this also not called start in the clients, and instead called "Schedule". Like this `await workflowClient.ScheduleNewWorkflowAsync`

Reply:
The `Schedule` method you're referring to isn't technically part of the management API. Rather, it's an API that's specific to just the Dapr engine. The actual Dapr client API in the .NET SDK is also named "Start". We'll be able to update this sample to use the general API instead of the engine-only API once we complete https://github.com/dapr/dotnet-sdk/issues/1027.

Reply:
I see. Because of the workaround. I raised this QS issue also https://github.com/dapr/quickstarts/issues/830 for v1.11